batches_test.go

```diff
@@ -0,0 +1,67 @@
+package packngo
+
+import (
+	"fmt"
+	"testing"
+)
+
+var batchID string
+
+func TestAccCreateBatch(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+	c := setup(t)
+
+	batches := &InstanceBatchCreateRequest{
+		Batches: []BatchInstance{
+			{
+				Hostname:        "test1",
+				Description:     "test batch",
+				Plan:            "baremetal_0",
+				OperatingSystem: "coreos_stable",
+				Facility:        "ewr1",
+				BillingCycle:    "hourly",
+				Tags:            []string{"abc"},
+				Quantity:        1,
+			},
+		},
+	}
+
+	batch, _, err := c.Batches.Create("93125c2a-8b78-4d4f-a3c4-7367d6b7cca8", batches)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if batch != nil {
+		batchID = batch.ID
+	}
+}
+func TestAccListBatches(t *testing.T) {
```

Comment:
Maybe test the List and Get methods in the same fucntion as the Create is tested. I would see it as:

1. Create batch with *n* device
2. Verify that the create request returned *n* items.
3. Wait for all the devices to be active.
4. Test the List function, and check that it returns *n* devices in active state.
5. Get the *n-1*th device (to test the Get)
5. Issue delete action
6. Check that all the devices are deleted.

Reply:
Will do